lib/components/helpers/pretty-text-input.js

```diff
@@ -111,7 +112,13 @@ export default createReactClass({
       if (pos) {
         this.codeMirror.replaceRange(tag, {line: pos.line, ch: pos.start}, {line: pos.line, ch: pos.stop});
       } else {
+        if (this.state.cursorPosition) {
+          this.codeMirror.setCursor(this.state.cursorPosition);
```

Comment:
It seems odd to me to only set the position when picking a choice. Seems like we should set the position right after creating the new codemirror instance. See my other comment.